src/adapters/deno.ts

```diff
@@ -25,7 +30,9 @@ class DenoServer implements Server<DenoFetchHandler> {
   constructor(options: ServerOptions) {
     this.options = options;
 
-    const fetchHandler = wrapFetch(this, this.options.fetch);
+    let fetchHandler = this.options.fetch;
+    fetchHandler = wrapFetchOnError(this.options.fetch, this.options.onError);
+    fetchHandler = wrapFetch(this, fetchHandler);
```

Comment:
Deno supports `onError` as a server option. We don't need to wrap fetch.

Reply:
Addressed in https://github.com/unjs/srvx/pull/48/commits/6353f4c5430581ea5d321f9481ddea741e833f51